polkadot-protocol/protocol-components/accounts/address-formats.md

```diff
@@ -0,0 +1,183 @@
+
+There are two types of checksum preimages (`SS58` and `AccountID`) along with a range of checksum lengths from 1 to 8 bytes.
+
+In all cases for Polkadot SDK, the Blake2b-512 hash function is used. The variants simply select the preimage used as the input to the hash function and the number of bytes taken from its output.
+
+The bytes used are always the left-most bytes. The input consists of the non-checksum portion of the SS58 byte series, which is used as input to the base-58 function. 
+
+For example, this input is formed by concatenating the `concat ( <address-type>, <address> )`. A context prefix of `0x53533538505245` (the string "SS58PRE") is then prepended to this input to create the final hashing preimage.
+
+The advantage of using additional checksum bytes lies in the increased protection they offer against input errors and index alterations. However, this comes at the expense of expanding the textual address by a few extra characters. 
+
+In the case of the account ID format, this increase is negligible, which is why no 1-byte alternative is provided. For shorter account-index formats, the extra byte constitutes a significantly larger portion of the final address. 
+
+Therefore, it is left to higher levels of the stack (rather than the end user) to determine the best trade-off based on their specific needs.
+
+## Address Types and Network Registry
+
+The [SS58 registry](https://github.com/paritytech/ss58-registry){target=\_blank} is the canonical listing of all address type identifiers and how they map to Polkadot SDK-based networks.
+
+## Encoding Address and Network Identifiers
+
+Identifiers up to the value of 64 can be represented using a simple address format, where the least significant byte of the network identifier is encoded as the first byte of the address.
+
+For identifiers ranging from `64` to `16,383`, the full address format must be utilized.
+
+The full address encoding requires special handling because SCALE encoding as little endian requires the first two bits to be used for the `01` prefix. 
+
+To encode the network identifier, the full address format treats the first two bytes as a `16-bit` sequence, and disregards the first two bits of that sequence to account for the `01` prefix. 
+
+The remaining 14 bits encode the network identifier value as little endian, with the assumption that the two missing higher order bits are zero. This effectively spreads the low-order byte across the boundary between the two bytes.
+
+For example, the 14-bit identifier `0b00HHHHHH_MMLLLLLL` is expressed in two bytes as:
+
+```
+0b01LLLLLL
+0bHHHHHHMM
+```
+!!! note 
+    Identifiers of `16384` and beyond aren't currently supported.
+
+## Validating Addresses
+
+You can verify that a value is a valid `SS58` address by using the subkey `inspect command` or though the `Polkadot-JS API`.
+
+### Using subkey
+
+!!!note "What is Subkey?"
```

Comment:
I'm not sure if we should use a note for explaining what it is or just a paragraph

Reply:
Just an opening sentence like "The subkey program is a key generation and management utility that is included in the Polkadot SDK repository.", then instructions is fine